### ISSUE TITLE
Can't send xrp, no tag

### ISSUE THREAD

**Issue by Capegitgirl** [sologenic/sologenic-expo-wallet#88]
Can't send xrp to sologenic without tag!  What is tag?

**Comment by bdg121** [comment]
I can get my wallet to set a tag. It says “processing” but won’t complete. I can’t send XRP to my wallet without a tag and SOLOGENIC wallet won’t create a tag

**Comment by rundy22** [comment]
Me too

**Comment by Hazy1369** [comment]
> Can't send xrp to sologenic without tag! What is tag?

Just put in a 1. I did this when I sent Solo from Hotbit. Without a number I couldn't withdraw. Anyway, I got lucky & it worked. Hobbit took 15% fee for withdrawing so I don't recommend them.

**Comment by Hazy1369** [comment]
That was Hotbit.

**Comment by aftaabjariwala** [comment]
Same issue, the process is in a time loop and have to shutdown the app to use it again

**Comment by ihsangunev** [comment]
Same here. App stucks when trying to activating "receiver tag"

**Comment by Cryptomaxie** [comment]
Will solo dex support the air drop without setting up a trust line?

**Comment by Hazy1369** [comment]
No, set up a Xumm wallet & add a Solo wallet.  That should do it.

Sent from Yahoo Mail on Android 
 
  On Fri, Nov 19, 2021 at 7:44 AM, ***@***.***> wrote:   


Will solo dex support the air drop without setting up a trust line?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
